src/dist/mark/bouncycastle/RulesTR_Cipher.mark

```diff
@@ -353,3 +353,19 @@ rule ID_3_5_01 {
 //    onfail
 //        InsufficientRSAKeylength // FIXME valid until 2022
```

Comment:
Is this a FIXME that requires immediate action or simply a reminder? If so, change to // REMINDER